File: android/employer/app/src/main/java/com/workforcex/employer/api/JobResponse.java
package com.workforcex.employer.api;

// This class must exactly match the fields from the backend's JobResponse record.
public class JobResponse {
    public String id;
    public String employerId;
    public String title;
    public String skillsRequired;
    public Integer experienceRequired;
    public String location;
    public Double salaryMin;
    public Double salaryMax;
    public Integer openPositions;
    public String description;
}

File: android/employer/shared_employer/src/commonMain/kotlin/com/workforcex/shared_employer/models/JobResponse.kt
package com.workforcex.shared_employer.models

data class JobResponse(
    val id: String,
    val employerId: String,
    val title: String,
    val skillsRequired: String,
    val experienceRequired: Int,
    val location: String,
    val salaryMin: Double,
    val salaryMax: Double,
    val openPositions: Int,
    val description: String
)